### PR TITLE
feat: docs useContractRead

### PR DESCRIPTION
**Pull request by arthurgeron**
- Added doc for new [useContractRead](https://github.com/FuelLabs/fuels-npm-packs/pull/91) hook

image

### CODE REVIEW

File: packages/docs/docs/dev/hooks-reference.mdx [hunk 1]
```diff
@@ -5,6 +5,36 @@ category: For Developers
 
 # React Hooks Reference
 
+## `useContractRead`
+
+Reads and calls a method from a Fuel contract, returns `<InvokeFunctions>`.
+
+
+#### Reading with a Contract instance
```
Comment (helciofranco) [inline packages/docs/docs/dev/hooks-reference.mdx:13]: h5 looks better imo

```suggestion
##### Reading with a Contract instance
```

File: packages/docs/docs/dev/hooks-reference.mdx [hunk 2]
```diff
@@ -5,6 +5,36 @@ category: For Developers
...
+```tsx
+const { contract } = useContractRead({
+  contract: _contract,
+  functionName: 'get_count',
+  args: undefined,
+});
+```
+
+#### Reading with ABI + ContractId + Provider
```
Comment (helciofranco) [inline packages/docs/docs/dev/hooks-reference.mdx:22]: same

File: packages/docs/docs/dev/hooks-reference.mdx (hunk 1 above)
Comment (helciofranco) [reply packages/docs/docs/dev/hooks-reference.mdx:13]: I'll use h5 in my docs too (for those subsections)

Screenshot 2024-06-03 at 13 28 07

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
put the `useContractRead` in alphabetical order

File: packages/docs/docs/dev/hooks-reference.mdx (hunk 1 above)
Comment (arthurgeron) [reply packages/docs/docs/dev/hooks-reference.mdx]: Good one, fixed

File: packages/docs/docs/dev/hooks-reference.mdx [hunk 3]
```diff
@@ -87,6 +87,34 @@ console.log(connectors);
 
 [See the source file](https://github.com/FuelLabs/fuels-npm-packs/tree/main/packages/react/src/hooks/useConnectors.ts)
 
+## `useContractRead`
+
+Reads and calls a method from a Fuel contract, returns `<InvokeFunctions>`.
+
+
+### Reading with a Contract instance
```
Comment (helciofranco) [inline packages/docs/docs/dev/hooks-reference.mdx:95]: imo h5 looks better, what you think?

```suggestion
##### Reading with a Contract instance
```

File: packages/docs/docs/dev/hooks-reference.mdx [hunk 4]
```diff
@@ -87,6 +87,34 @@ console.log(connectors);
...
+```tsx
+const { contractRead } = useContractRead({
+  contract: _contract,
+  functionName: 'get_count',
+  args: undefined,
+});
+```
+
+### Reading with ABI + ContractId + Provider
```
Comment (helciofranco) [inline packages/docs/docs/dev/hooks-reference.mdx:104]: Same here

```suggestion
##### Reading with ABI + ContractId + Provider
```

**Review by LuizAsFight — APPROVED** [review]

**Pull request merged**